Automattic/wp-calypso
# Bulk domain transfer flow: Update copy to recommended strings.

Related to https://github.com/Automattic/dotcom-forge/issues/2987

## Proposed Changes

* Update copy.

Before | After
------ | ------
<img width="586" alt="Screen Shot 2023-07-10 at 3 36 00 PM" src="https://github.com/Automattic/wp-calypso/assets/1689238/65c3e445-3af1-470f-9573-80b53b9fec98"> | <img width="651" alt="Screen Shot 2023-07-10 at 3 35 02 PM" src="https://github.com/Automattic/wp-calypso/assets/1689238/d7575225-378a-4e94-8914-566c5e74aed7">
<img width="802" alt="Screen Shot 2023-07-10 at 3 35 50 PM" src="https://github.com/Automattic/wp-calypso/assets/1689238/82fa1758-f3a9-46a2-a024-1bbb029cc394"> | <img width="814" alt="Screen Shot 2023-07-11 at 12 05 26 PM" src="https://github.com/Automattic/wp-calypso/assets/1689238/48496f39-6ee7-4bbe-83e4-c97257e7abfd">


## Testing Instructions



* Check that copy is updated on Intro and Complete pages in the bulk domain transfer flow.

## Pre-merge Checklist
